### DISCUSSION TITLE
Show Only On Desktop And Password Protection to Settings Window

### DISCUSSION THREAD

**Discussion by AydinFatoglu** [Flow-Launcher/Flow.Launcher#1385] [Ideas]
I have an idea / request ,

1) is there a way to make a setting to always show the search box ONLY on desktop only? 
2) Settings window password lock to avoid end user to change the settings that set by system admins

**Comment by Garulf** [comment]
> 2) Settings window password lock to avoid end user to change the settings that set by system admins

You could potentially set the `settings.json` to read only for the user in question

**Reply by AydinFatoglu** [reply]
it brakes the software

**Reply by jjw24** [reply]
Probably can achieve this by having settings to require administrator access, which saves from implementing password protection.

**Reply by jjw24** [reply]
What settings do you require to protect out of interest?

**Comment by jjw24** [comment]
>1. is there a way to make a setting to always show the search box ONLY on desktop only?

What would be the use case for this ?

**Reply by AydinFatoglu** [reply]
on vdi / kiosk pc environments . WHİCH we deploy to regular temporary users we don't want user to change or set configuration on the programs.

The search bar's being on top off all windows kind of blocks the usage

**Reply by jjw24** [reply]
Ok so for this you then could allow the user to search things using when the desktop is shown. Not sure how difficult it is to detect desktop.

**Comment by onesounds** [comment]
Both are features that do not currently exist. We can make it, but I think it's more of a job (customize for business)  where we have to get paid because it's not for the average user. Even if we make on it, I'm not sure this is the specification you really want.
